### DISCUSSION TITLE
Different accuracy in Paper and Github page?

### DISCUSSION THREAD

**Discussion by karthickpgunasekaran** [thunlp/FewRel#57] [General]
Hello there,
   I noticed that there is a huge difference between the accuracies mentioned in the paper and the ones on the Github page mainly wrt to Fewrel 2.0 Domain adaptations. Can you please help explain on why this discrepancy is present? Thanks !

**Comment by gaotianyu1350** [comment]
Hi,

Which one were you referring to? If you are talking about BERT-PAIR, yes, there is a big difference. After publishing the paper, we found that we can further improve the performance by better tuning the hyper-parameters, so we updated the better results we get here.
